dev.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+"""
+A tool to support the maintenance of templates and examples in this repository.
+Something like a Makefile but written in Python for easier maintenance.
+
+To list the available commands, run ./dev.py --help.
+"""
+import argparse
```

Comment:
FYI, if you're looking for other make alternatives, I really like https://taskfile.dev/